.gitignore

```diff
@@ -24,6 +24,7 @@ bld/
 [Oo]bj/
 [Ll]og/
 [Oo]ut/
+.artifactsCache
```

Comment:
This is the default value of the `NewDevEnv.ps1`, but what if someone's using a non-default cache path? Is it feasible to blanket exclude `*.app` files from being uploaded to source control? That's how the AL-Go templates handle it:
* https://github.com/microsoft/AL-Go-PTE/blob/main/.gitignore
* https://github.com/microsoft/AL-Go-AppSource/blob/main/.gitignore

Reply:
*.app files are already excluded (see line 305).

If someone decides to use a use a non-default cache path that is the repo, they'll need to bear the consequences. The cache folder doesn't store only the .app files anyway.